.prettierrc.json

```
@@ -0,0 +1,20 @@
+{
+    "printWidth": 140,
+    "tabWidth": 4,
+    "useTabs": false,
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "none",
+    "bracketSpacing": true,
+    "bracketSameLine": false,
+    "proseWrap": "always",
+    "arrowParens": "avoid",
```

could you add as well the sort imports plugin while at it ?


```suggestion
    "arrowParens": "avoid",
    "plugins": ["@trivago/prettier-plugin-sort-imports"],
    "importOrder": ["<THIRD_PARTY_MODULES>", "^../", "^./"],
    "importOrderSortSpecifiers": true,
    "importOrderSeparation": true,
```

---

There's an eslint plugin for that I believe rather than prettier. 

I also suggest introducing changes in an iterative matter, more can be done after this PR.